VAMobile/src/components/FormWrapper/FormFields/VASelector.test.tsx

```diff
@@ -2,12 +2,11 @@ import 'react-native'
 import React from 'react'
 // Note: test renderer must be required after react-native.
 import 'jest-styled-components'
```

Comment:
should be able to remove the import from line 4

Reply:
Updated.